classes/image_type_tezi_ot3.bbclass

```diff
@@ -185,8 +185,8 @@ def rootfs_tezi_emmc(d, use_bootfiles):
     
     filesystem_partitions.append(
           {
-            "partition_size_nominal": 512,
-            "want_maximised": True,
+            "partition_size_nominal": 3000,
```

Comment:
ditto with the power of 2 - 3072 i believe is what it is. makes me twitch haha

Reply:
> Let's do powers of two here but otherwise looks good. What does `want_maximized` do?

That lets partitions expand and fill up free space on the emmc. Was getting weird sizes, so turned that to false.